data/campaigns/World_Conquest/resources/strings/effects.cfg

```diff
@@ -82,10 +82,10 @@ _ "terrain^water" #enddef
 _ "improved defenses" #enddef
 
 #define STR_LIVING
-_ "no undead^consumed by a living beign:" #enddef
+_ "no undead^consumed by a living being:" #enddef
 
 #define STR_UP_TO VALUE
-_ "up to"+" {VALUE}% " #enddef
+_ "up to"+" {VALUE}% " #enddef
```

Comment:
Whats the reason for this change ?

Reply:
@stevecotton remarked it stood out. What's the reason for the non-break?

Reply:
To make the dialog where this text appears in looks better,  there are quite a few other places where this addons uses nonbreaking strings, and it does that intentionally. ( https://github.com/gfgtdf/World_Conquest_II/commit/6c352fb216853308b1b8c71e5870a1675ca0a889 ) .

Reply:
Actually i'm not 100% sure where  utils/gameplay/training.cfg  still has them becasue i changed tehse strigns at some point and i migth ahve accidentatlyl dropped them here (which  which be bad because that's the main place where they are needed.)